src/pages/registration/resource_type_tab/sub_type_forms/research_data_types/RelatedResultItem.tsx

```diff
@@ -88,60 +80,54 @@ export const RelatedResultItem = ({
-                    registration?.identifier ?? ''
-                  )}
-                  component={RouterLink}
-                  to={getRegistrationLandingPagePath(registration?.identifier ?? '')}>
-                  {getTitleString(registration?.entityDescription?.mainTitle)}
-                </Link>
-              ) : (
-                <Link data-testid={dataTestId.registrationWizard.resourceType.externalLink} href={uri}>
-                  {uri}
-                </Link>
-              )}
-            </Box>
+            <SearchListContainer sx={{ borderLeftColor: 'registration.main', width: '100%' }}>
+              <RegistrationListItemContent
+                registration={convertToRegistrationSearchItem(registration)}
+                onRemoveRelated={() => setIndexToRemove(index)}
+              />
+            </SearchListContainer>
           )}
         </>
       ) : (
-        <Field name={`${ResourceFieldNames.PublicationInstanceRelated}[${index}].text`}>
-          {({ field, meta: { touched, error } }: FieldProps<string>) => (
-            <TextField
-              {...field}
-              label={t('common.reference')}
-              placeholder={t('feedback.validation.reference_required')}
-              variant="filled"
-              multiline
-              fullWidth
-              required
-              error={touched && !!error}
-              helperText={<ErrorMessage name={field.name} />}
-            />
-          )}
-        </Field>
-      )}
-
-      <IconButton
-        title={t('registration.resource_type.research_data.remove_relation')}
-        data-testid={dataTestId.registrationWizard.resourceType.removeRelationButton(index.toString())}
-        onClick={() => setIndexToRemove(index)}>
-        <CloseIcon
-          fontSize="small"
+        <Box
           sx={{
-            color: 'white',
-            borderRadius: '50%',
-            bgcolor: 'primary.main',
-          }}
-        />
-      </IconButton>
+            display: 'flex',
+            gap: '1rem',
+            bgcolor: 'white',
+            borderRadius: '0.25rem',
+            border: '1px solid lightgray',
+            p: '0.5rem',
+            width: '100%',
+          }}>
+          <Field name={`${ResourceFieldNames.PublicationInstanceRelated}[${index}].text`}>
+            {({ field, meta: { touched, error } }: FieldProps<string>) => (
+              <TextField
+                {...field}
+                label={t('common.reference')}
+                placeholder={t('feedback.validation.reference_required')}
+                variant="filled"
+                multiline
+                fullWidth
+                required
+                error={touched && !!error}
+                helperText={<ErrorMessage name={field.name} />}
+              />
+            )}
+          </Field>
+          <IconButton
+            sx={{ mr: '0.5rem' }}
```

Comment:
noen grunn til at boksen skal ha mer margin på høyresiden enn venstre?
```suggestion
```

Reply:
Er for å få den mer på linje med knappen på internt resultat.

uten margin:
![Screenshot 2024-11-15 at 13 43 23](https://github.com/user-attachments/assets/61fbb9d0-b596-40f8-8bda-74ffaecdeaaa)

med margin:
![Screenshot 2024-11-15 at 13 45 40](https://github.com/user-attachments/assets/8de48287-0fd7-4ed2-a661-b2525ca9b2de)

Reply:
Litt hacky er det jo å legge til en ekstra margin for å få det ish på linje, men greit nok for denne tenker jeg 😅 (om du ikke ser noen enkel generell løsning)